lib/process-services/src/lib/task-list/components/task-filters.component.ts

```diff
@@ -71,15 +73,29 @@ export class TaskFiltersComponent implements OnInit, OnChanges {
 
     filters: FilterRepresentationModel [] = [];
 
+    private onDestroy$ = new Subject<boolean>();
+    currentRoute: string = '';
+    isTaskRoute: boolean;
+
     private iconsMDL: IconModel;
 
     constructor(private taskFilterService: TaskFilterService,
                 private taskListService: TaskListService,
-                private appsProcessService: AppsProcessService) {
+                private appsProcessService: AppsProcessService,
+                private router: Router) {
     }
 
     ngOnInit() {
         this.iconsMDL = new IconModel();
+        this.router.events
+            .pipe(
+                filter((event) => event instanceof NavigationStart),
+                takeUntil(this.onDestroy$)
+            )
+            .subscribe((navigationStart: NavigationStart) => {
+                this.currentRoute = navigationStart.url;
```

Comment:
you can replace with local variable
```suggestion
                const currentRoute = navigationStart.url;
```

Reply:
Replaced